docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -302,75 +308,162 @@ role ``meta/argument_specs.yml`` file. All fields are lower-case.
+            * If missing, the option is not required.
 
         :default:
 
-           * If ``required`` is false/missing, ``default`` may be specified (assumed 'null' if missing).
-           * Ensure that the default value in the docs matches the default value in the code. The actual
-             default for the role variable will always come from ``defaults/main.yml``.
-           * The default field must not be listed as part of the description, unless it requires additional information or conditions.
-           * If the option is a boolean value, you should use `true/false` if you want to be compatible with `ansible-lint`.
+            * If ``required`` is false/missing, ``default`` may be specified (assumed 'null' if missing).
+            * Ensure that the default value in the docs matches the default value in the code. The actual
+              default for the role variable will always come from ``defaults/main.yml``.
+            * The default field must not be listed as part of the description, unless it requires additional information or conditions.
+            * If the option is a boolean value, you should use `true/false` if you want to be compatible with `ansible-lint`.
 
         :choices:
 
-           * List of option values.
-           * Should be absent if empty.
+            * List of option values.
+            * Should be absent if empty.
 
         :elements:
 
-           * Specifies the data type for list elements when type is ``list``.
+            * Specifies the data type for list elements when type is ``list``.
 
         :options:
 
-           * If this option takes a dict or list of dicts, you can define the structure here.
+            * If this option takes a dict or list of dicts, you can define the structure here.
 
 Sample specification
 --------------------
 
 .. code-block:: yaml
 
- # roles/myapp/meta/argument_specs.yml
- ---
- argument_specs:
-   # roles/myapp/tasks/main.yml entry point
-   main:
-     short_description: The main entry point for the myapp role.
-     options:
-       myapp_int:
-         type: "int"
-         required: false
-         default: 42
-         description: "The integer value, defaulting to 42."
-
-       myapp_str:
-         type: "str"
-         required: true
-         description: "The string value"
-
-   # roles/myapp/tasks/alternate.yml entry point
-   alternate:
-     short_description: The alternate entry point for the myapp role.
-     options:
-       myapp_int:
-         type: "int"
-         required: false
-         default: 1024
-         description: "The integer value, defaulting to 1024."
+  # roles/myapp/meta/argument_specs.yml
+  ---
+  argument_specs:
+    # roles/myapp/tasks/main.yml entry point
+    main:
+      short_description:
+        - The main entry point for the myapp role.
+      description:
+        - This is the main entrypoint for the C(myapp) role.
+        - Here we can describe what this entrypoint does in lenghty words.
```

Comment:
```suggestion
        - Here we can describe what this entrypoint does in lenghty words.
        - Every new list item is a new paragraph. You can have multiple sentences
          per paragraph.
```

Reply:
I will also put that hint in the description above. So in case somebody only reads the explanation of the `description` field and does not read this example.